src/current/v23.2/known-limitations.md

```diff
@@ -165,7 +138,17 @@ In cases where the partition definition includes a comparison with `NULL` and a
 
 {% include {{page.version.version}}/known-limitations/sql-cursors.md %}
 
-### `SELECT FOR UPDATE` locks are dropped on lease transfers  and range splits/merges
+### Limitations for `SELECT FOR UPDATE`
+
+- [`SELECT FOR UPDATE`]({% link {{ page.version.version }}/select-for-update.md %}) places locks on each key scanned by the base index scan. This means that even if some of those keys are later filtered out by a predicate which could not be pushed into the scan, they will still be locked.
```

Comment:
Like the limitation just below this one, I think it's worth mentioning that this limitation no longer exists either (a) when using read committed isolation or (b) with `optimizer_use_lock_op_for_serializable` set to true when using serializable isolation.

Maybe one way to do that would be to qualify it, like I tried below?

And I do think we should change the tracking link to cockroachdb/cockroach#114737

Reply:
Done, and using the wording that was published [here](https://github.com/cockroachdb/docs/blob/main/src/current/v23.2/known-limitations.md#limitations-for-for-update-and-for-share-locks-under-serializable).